src/vcpkg/build.cpp

```diff
@@ -263,14 +304,7 @@ namespace vcpkg::Build
             std::move(first_arg), default_triplet, COMMAND_STRUCTURE.example_text, paths);
 
         PathsPortFileProvider provider(paths, args.overlay_ports);
-        const auto port_name = spec.package_spec.name();
-        const auto* scfl = provider.get_control_file(port_name).get();
-
-        Checks::check_maybe_upgrade(VCPKG_LINE_INFO, scfl != nullptr, "Error: Couldn't find port '%s'", port_name);
```

Comment:
Since this PR is removing this error message, have you checked that the new error message along this path is still reasonable?

Reply:
I believe this error message was unreachable:

<img width="574" alt="image" src="https://user-images.githubusercontent.com/1544943/162814605-b03825cd-fc7c-4bbc-9405-2c1726caa9c9.png">